src/w3iProxy/notifyProviders/externalNotifyProvider.ts

```diff
@@ -74,14 +74,9 @@ export default class ExternalNotifyProvider implements W3iNotifyProvider {
     }
   }
 
-  public async register(params: { account: string; domain: string; isLimited?: boolean }) {
-    return this.postToExternalProvider('register', {
-      account: params.account,
-      isLimited: params.isLimited,
-      domain: params.domain,
-      // Signing will be handled wallet-side.
-      onSign: async () => Promise.resolve('')
-    })
+  public async register() {
+    // TODO: remove this whole provider
```

Comment:
Can we just remove `externalNotifyProvider` if it's now unused instead of setting a TODO or are there concerns around doing this?

Reply:
It would require a lot of code changes and refactors, not the place for it.